autoscale-scaler-kubernetes/pom.xml

```diff
@@ -60,5 +60,15 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Think we can remove these deps now as well, thanks

Reply:
Just getting caught up. I think the orginal Unit tests did not add much value. But there is still value is adding Unit tests. Can tests not be added that mock the AuthorizationV1Api responses, and so exercise the health check code?

Reply:
@frankmccarry I have been unable to mock AuthorizationV1Api because it is instantiated inside of the method. 
The effort associated with getting test running seems to outweigh the simple function that we are actually trying to test. Going to leave out unit tests and test on actual system.

Reply:
Refactored unit tests so that they are exercising the health check code correctly